.github/workflows/continuous-deployment.yml

```diff
@@ -96,6 +96,54 @@ jobs:
           path: gui/artifacts/
           if-no-files-found: error
 
+  pa11y-test:
+    defaults:
+      run:
+        working-directory: gui
+
+    runs-on: ubuntu-latest
+
+    needs: [ Build-Info ]
+    environment: ${{ needs.Build-Info.outputs.environment }}
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - name: Install chrome
+        uses: browser-actions/setup-chrome@v1
+
+      - name: Get source code
+        uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+          cache-dependency-path: gui/package-lock.json
+
+      - name: Install Node dependencies
+        run: npm ci && npm install -g pa11y-ci
+
+      - name: Run pa11y Test
+        run: |
+          set -e
```

Comment:
Note that there was a shell script, but it was an experiment and didn't work. That was before @magile32 figured out that we needed to install chrome on the runner. So it should work to move everything there.

Reply:
right, definitely don't need to run it in USTP env. Thought it would make workflow a little easier to read/manage if that portion of the job is stored in another file. Not a big deal, just a question